solutions/project/spoke-project-unclass/network/firewall.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Firewall
+## Enable traffic on mgmt network for fortigates HA
```

Comment:
add a firewall rule that affects all resources of the vpc to allow egress to the PSC endpoint. This rule needs to have a higher priority than the default deny

Reply:
done